Simulation.cpp

```diff
@@ -317,7 +317,7 @@ void Simulation::UpdateRoutesAndLocations()
 //    }
 
 #pragma omp parallel for shared(pedsToRemove, allRooms)
-     for (signed int p = 0; p < allPeds.size(); ++p) {
+     for (long unsigned int p = 0; p < allPeds.size(); ++p) {
```

Comment:
This is probably better replaced by a range based for loop
```cpp
for(auto& ped : allPeds){
...
}
```

Reply:
Apparently it does not work with clang and OpenMP 3.1 but compiles with gcc and OpenMP 4.5 (see Travis). From the standard I’m not entirely sure if it also distributes the computations across the cores or if it only compiles, since range based loop seem only be support with OpenMP 5.0

Reply:
I see, I did also some googling and it seems that it should (TM) work if it passes the compiler but I could not find a clear answer :/ Good call to leave it at this form. @chraibi Should there be a min OpenMP version specified?

Reply:
unless there is a very good reason for requiring a minimum OpenMP version, I would not do it.